app/helpers/application_helper/button/orchestration_template_edit_remove.rb

```diff
@@ -1,15 +1,16 @@
 class ApplicationHelper::Button::OrchestrationTemplateEditRemove < ApplicationHelper::Button::Basic
+  needs_record
+
+  def disabled?
+    @record.in_use?
+  end
+
   def calculate_properties
     super
-    if @view_context.x_active_tree == :ot_tree && @record
-      if @record.in_use?
-        self[:enabled] = false
-        self[:title] = if self[:id] =~ /_edit$/
-                         N_('Orchestration Templates that are in use cannot be edited')
-                       else
-                         N_('Orchestration Templates that are in use cannot be removed')
-                       end
-      end
-    end
+    self[:title] = if self[:id] =~ /_edit$/
```

Comment:
@jzigmund I think this should be assigned to `self[:title]` only if `disabled?` returns `true`. It seems to me that this could lead to a situation where you would have enabled button with label saying you can not do the operation.